_pkgdown.yml

```diff
@@ -1,12 +1,29 @@
-destination: docs
+url: https://atorus-research.github.io/xportr
 
 template:
+  bootstrap: 5
   params:
-    bootswatch: yeti
-  opengraph:
-    image:
-      src: man/figures/xportr_rev.png
-      alt: "xportr Hex Sticker"
+    bootswatch: sandstone
+search:
+  exclude: ['news/index.html']
+repo:
+  url:
+    home: https://github.com/atorus-research/xportr
+    source: https://github.com/atorus-research/xportr/blob/main
+    issue: https://github.com/atorus-research/xportr/issues
+    user: https://github.com/
```

Comment:
Not sure if I'm missing something but when I removed the `repo` nothing changed in the pkgdown site. If this is not doing anything, can we remove it?

Reply:
Looks like this is overrides for links found in the DESCRIPTION: https://pkgdown.r-lib.org/reference/build_site.html?q=repo#source-repository. 

Doesn't look necessary to me but I don't have strong opinions.